features/_trustedboot/convert.raw~esp.tar

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+input="$1"
+output="$2"
+
+esp_dir="$(mktemp -d)"
+mount -t tmpfs -o size="$TEMPFS_SIZE" tmpfs "$esp_dir"
```

Comment:
Where do we set `TEMPFS_SIZE`?

Reply:
builder

Reply:
[builder](https://github.com/gardenlinux/builder/blob/357fbe01a5a95854ccb5065c28fc6eca13466a46/build#L121)